plugins/optimization-detective/class-od-tag-visitor-registry.php

```diff
@@ -37,9 +37,20 @@ final class OD_Tag_Visitor_Registry implements Countable, IteratorAggregate {
 	 *
 	 * @param string   $id                   Identifier for the tag visitor.
 	 * @param callable $tag_visitor_callback Tag visitor callback.
+	 * @param array<mixed> $dependencies array with ID as key and Callable as value
```

Comment:
```suggestion
	 * @param string[] $dependencies Tag visitors that must run before this tag visitor.
```

Reply:
I feel this should be any so we can have more than one dependency

Reply:
Yes, and that's still the case: `string[]` is an array of strings.

Reply:
changed

Reply:
I don't see the change.